addons/sale/views/variant_templates.xml

```diff
@@ -17,7 +18,7 @@
                     <t t-if="ptal.attribute_id.display_type == 'select'">
                         <select
                             t-att-data-attribute_id="ptal.attribute_id.id"
-                            t-attf-class="custom-select css_attribute_select js_variant_change #{ptal.attribute_id.create_variant} #{'d-none' if single_and_custom else ''}"
+                            t-attf-class="custom-select css_attribute_select product_attribute js_variant_change #{ptal.attribute_id.create_variant} #{'d-none' if single_and_custom else ''}"
```

Comment:
```suggestion
                            t-attf-class="custom-select css_attribute_select o_wsale_product_attribute js_variant_change #{ptal.attribute_id.create_variant} #{'d-none' if single_and_custom else ''}"
```
Please try to follow the convention, it makes it easier to figure which classes are related to odoo.
I am not too sure of what it exactly is, `o_wsale_` or `o_` or `oe_`, but at least something to better identify it

(You might wanna change it now that you handle the module, but probably keep something in that regarde)